<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsComposerLegendWidgetBase</class>
 <widget class="QWidget" name="QgsComposerLegendWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Barscale Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>405</width>
        <height>490</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>362</width>
            <height>647</height>
           </rect>
          </property>
          <attribute name="label">
           <string>General</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="7" column="1">
            <widget class="QPushButton" name="mTitleFontButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Title Font...</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QPushButton" name="mGroupFontButton">
             <property name="text">
              <string>Group Font...</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QPushButton" name="mLayerFontButton">
             <property name="text">
              <string>Layer Font...</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QPushButton" name="mItemFontButton">
             <property name="text">
              <string>Item Font...</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QDoubleSpinBox" name="mSymbolWidthSpinBox">
             <property name="prefix">
              <string>Symbol width </string>
             </property>
             <property name="suffix">
              <string> mm</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QDoubleSpinBox" name="mSymbolHeightSpinBox">
             <property name="prefix">
              <string>Symbol height </string>
             </property>
             <property name="suffix">
              <string> mm</string>
             </property>
            </widget>
           </item>
           <item row="15" column="1">
            <widget class="QDoubleSpinBox" name="mLayerSpaceSpinBox">
             <property name="prefix">
              <string>Layer space </string>
             </property>
             <property name="suffix">
              <string> mm</string>
             </property>
            </widget>
           </item>
           <item row="17" column="1">
            <widget class="QDoubleSpinBox" name="mSymbolSpaceSpinBox">
             <property name="prefix">
              <string>Symbol space </string>
             </property>
             <property name="suffix">
              <string> mm</string>
             </property>
            </widget>
           </item>
           <item row="18" column="1">
            <widget class="QDoubleSpinBox" name="mIconLabelSpaceSpinBox">
             <property name="prefix">
              <string>Icon label space </string>
             </property>
             <property name="suffix">
              <string> mm</string>
             </property>
            </widget>
           </item>
           <item row="19" column="1">
            <widget class="QDoubleSpinBox" name="mBoxSpaceSpinBox">
             <property name="prefix">
              <string>Box space </string>
             </property>
             <property name="suffix">
              <string> mm</string>
             </property>
            </widget>
           </item>
           <item row="26" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="22" column="1">
            <widget class="QComboBox" name="mMapComboBox"/>
           </item>
           <item row="21" column="1">
            <widget class="QLabel" name="mMapLabel">
             <property name="text">
              <string>Map</string>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="QDoubleSpinBox" name="mGroupSpaceSpinBox">
             <property name="prefix">
              <string>Group Space </string>
             </property>
             <property name="suffix">
              <string> mm</string>
             </property>
            </widget>
           </item>
           <item row="24" column="1">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Wrap text on</string>
             </property>
            </widget>
           </item>
           <item row="25" column="1">
            <widget class="QLineEdit" name="mWrapCharLineEdit">
             <property name="frame">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="mTitleLabel">
               <property name="text">
                <string>&amp;Title</string>
               </property>
               <property name="buddy">
                <cstring>mTitleLineEdit</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="mTitleLineEdit"/>
             </item>
            </layout>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="mColumnCountSpinBox">
             <property name="prefix">
              <string>Column count </string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QCheckBox" name="mSplitLayerCheckBox">
             <property name="toolTip">
              <string>Allow to split layer items into multiple columns.</string>
             </property>
             <property name="text">
              <string>Split layers</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QCheckBox" name="mEqualColumnWidthCheckBox">
             <property name="text">
              <string>Equal column widths</string>
             </property>
            </widget>
           </item>
           <item row="20" column="1">
            <widget class="QDoubleSpinBox" name="mColumnSpaceSpinBox">
             <property name="prefix">
              <string>Column space </string>
             </property>
             <property name="suffix">
              <string> mm</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QPushButton" name="mFontColorPushButton">
             <property name="text">
              <string>Font color...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>429</width>
            <height>385</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Legend items</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" colspan="4">
            <widget class="QCheckBox" name="mCheckBoxAutoUpdate">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Auto Update</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="10">
            <widget class="QTreeView" name="mItemTreeView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ContiguousSelection</enum>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QToolButton" name="mMoveDownToolButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../images/images.qrc">
               <normaloff>:/images/themes/default/mActionArrowDown.png</normaloff>:/images/themes/default/mActionArrowDown.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QToolButton" name="mMoveUpToolButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../images/images.qrc">
               <normaloff>:/images/themes/default/mActionArrowUp.png</normaloff>:/images/themes/default/mActionArrowUp.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QToolButton" name="mAddToolButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../images/images.qrc">
               <normaloff>:/images/themes/default/mActionSignPlus.png</normaloff>:/images/themes/default/mActionSignPlus.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QToolButton" name="mRemoveToolButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../images/images.qrc">
               <normaloff>:/images/themes/default/mActionSignMinus.png</normaloff>:/images/themes/default/mActionSignMinus.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QToolButton" name="mEditPushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../images/images.qrc">
               <normaloff>:/images/themes/default/symbologyEdit.png</normaloff>:/images/themes/default/symbologyEdit.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="6">
            <widget class="QToolButton" name="mUpdatePushButton">
             <property name="text">
              <string>Update</string>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="7">
            <widget class="QToolButton" name="mUpdateAllPushButton">
             <property name="text">
              <string>All</string>
             </property>
            </widget>
           </item>
           <item row="2" column="9">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="8">
            <widget class="QToolButton" name="mAddGroupButton">
             <property name="text">
              <string>Add group</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QToolButton" name="mCountToolButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Show feature count for each class of vector layer.</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../images/images.qrc">
               <normaloff>:/images/themes/default/mActionSum.png</normaloff>:/images/themes/default/mActionSum.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>mTitleFontButton</tabstop>
  <tabstop>mGroupFontButton</tabstop>
  <tabstop>mLayerFontButton</tabstop>
  <tabstop>mItemFontButton</tabstop>
  <tabstop>mSymbolWidthSpinBox</tabstop>
  <tabstop>mSymbolHeightSpinBox</tabstop>
  <tabstop>mLayerSpaceSpinBox</tabstop>
  <tabstop>mSymbolSpaceSpinBox</tabstop>
  <tabstop>mIconLabelSpaceSpinBox</tabstop>
  <tabstop>mBoxSpaceSpinBox</tabstop>
  <tabstop>mItemTreeView</tabstop>
  <tabstop>mMoveDownToolButton</tabstop>
  <tabstop>mMoveUpToolButton</tabstop>
  <tabstop>mRemoveToolButton</tabstop>
  <tabstop>mEditPushButton</tabstop>
  <tabstop>mUpdatePushButton</tabstop>
  <tabstop>mUpdateAllPushButton</tabstop>
  <tabstop>mAddGroupButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
